doc/releases/release-notes-3.2.rst

```diff
@@ -609,6 +633,11 @@ HALs
 
 * NXP
 
+  * Updated the NXP MCUX SDK to version 2.12
+  * Update the USB middleware to version 2.12
+  * Remove all binary Blobs for power management libraries
+  * Remove all binary archive files
```

Comment:
Can you keep these in the past? (Updated, Removed...)

Reply:
Changed.